### DISCUSSION TITLE
LinkedIn Auth

### DISCUSSION THREAD

**Discussion by richiemcilroy** [supabase/supabase#2593] [Feature Requests]
I went to add a LinkedIn sign in button yesterday and realised that it's not an auth option - can this be added? Or is this something that I could potentially create myself and submit as a pull request?

**Comment by inian** [comment]
This is not on the roadmap yet @richiemcilroy, but Pull requests welcome :) For example, [this](https://github.com/supabase/gotrue/pull/135) is how the Twitch provider was implemented which should be a good starting point

**Reply by FlorianBouron** [reply]
I have a question about that. If we do implement it and it goes merged when could it be available on the dashboard?

**Reply by MildTomato** [reply]
Do you mean as an auth provider for you own projects, or as an actual auth login option for you to log into Supabase dashboard?

**Reply by FlorianBouron** [reply]
What I mean is if it would be available on the sass version of the dashboard? Not on the self hosted.

**Reply by MildTomato** [reply]
Yea we can add it as a provider as soon as it’s added to gotrue and gotrue-js libraries

**Comment by FlorianBouron** [comment]
> I went to add a LinkedIn sign in button yesterday and realised that it's not an auth option - can this be added? Or is this something that I could potentially create myself and submit as a pull request?

Did you start any PR about this?
I am interested in it as well.

**Reply by MildTomato** [reply]
Doesn't look like it - repo is [here](https://github.com/supabase/gotrue) if you're interested

**Reply by MildTomato** [reply]
issue —> https://github.com/supabase/gotrue/issues/61

**Comment by MildTomato** [comment]
LinkedIn is now a provider option in supabase ⚡️
